VERSION

```diff
@@ -1 +1 @@
-3.1.1
\ No newline at end of file
+3.1.18.1
```

Comment:
could we stick to 3 digits versioning?
I tried to used the following convention: if version is A.B.C, A=Major changes (w/ potential compatibility problems).  B=Add on features. C=Bug fixes and small changes